src/dispatch/plugins/dispatch_slack/incident/interactive.py

```diff
@@ -806,7 +806,7 @@ def handle_timeline_added_event(
 
         source = "Slack message"
         # if the individual is not found, see if it is a bot
-        if individual is None:
+        if not individual:
```

Comment:
It may still be undefined here, we need to set `individual` to `None` outside the `except`